test/support/page_objects/dashboard_page.js

```diff
@@ -276,8 +319,19 @@ export default class DashboardPage {
     await PageObjects.header.setAbsoluteRange(fromTime, toTime);
   }
 
-  async storeTimeWithDashboard(on) {
+  async saveAsNewCheckbox(on) {
```

Comment:
Can we rename this to `setSaveAsNewCheckBox(checked)` to make it clear what the action is that we're taking?